de_workloads/shared_resources/data_factory/vars.tf

```diff
@@ -0,0 +1,66 @@
+variable "data_factory" {
+  type        = string
+  default     = "amido-stacks-dev-euw-rs"
+  description = "Azure Data Factory name"
+}
+
+variable "data_factory_resource_group_name" {
+
+  type        = string
+  default     = "amido-stacks-dev-euw-de"
+  description = "Azure Data Factory resource group name"
+}
+
+variable "key_vault_resource_group_name" {
+
+  type        = string
+  default     = "amido-stacks-dev-euw-de"
+  description = "Azure Key Vault resource group name"
+}
+
+variable "key_vault_uri" {
+  type        = string
+  description = "Azure Key Vault URI"
+}
+
+variable "integration_runtime_name" {
+  type        = string
+  description = "Azure Data Factory Integration Runtime name"
+}
+
+variable "blob_configstore_name" {
```

Comment:
I am actually thinking more to take the defaults out of these files, as these values are required and will be unique with each deployment.

Reply:
I find value in those defaults because they make it easier to understand what type of parameter value is required - it is not always obvious having only the variable name and description. Maybe we could have examples in the description? Or some other place? :)